ecommerce/api.py

```diff
@@ -458,12 +458,19 @@ def enroll_user_in_order_items(order):
         )
         edx_request_success = False
 
+    voucher = order.purchaser.vouchers.order_by("uploaded").last()
+    voucher_target = None
+    if voucher and not voucher.is_redeemed() and voucher.product is not None:
+        voucher_target = voucher.product.content_object
     for run in runs:
-        CourseRunEnrollment.objects.get_or_create(
+        enrollment = CourseRunEnrollment.objects.get_or_create(
             user=order.purchaser,
             run=run,
             defaults=dict(company=company, edx_enrolled=edx_request_success),
-        )
+        )[0]
```

Comment:
instead of using `[0]` you can do `enrollment, _ = ` at the beginning of this statement